src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+    private static final int OPERATION_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = createReport(fromFileName, toFileName);
+        writeReportToFile(report, toFileName);
+    }
+
+    private String createReport(String fromFileName, String toFileName) {
```

Comment:
you should move logic of reading from file to a separate method

Reply:
I renamed the method to read the file, create a report through StringBuffer, I can’t complain that I need to change it to String. Explain more precisely what you expect from me, I do not quite understand what you want from me.